### PR TITLE
Clean up readme

### PR DESCRIPTION
**Pull request by nbarbettini**
Small clean up to the d.o.c github readme:
* Removed broken or out of date links
* Added more prominent link to Dev Forum
* Made it clear that "Get Started" is really about building locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,25 @@
 <p align="center">
-<img src="https://devforum.okta.com/uploads/oktadev/original/1X/bf54a16b5fda189e4ad2706fb57cbb7a1e5b8deb.png" href='https://devforum.okta.com/' width="512px"/>
+<img src="https://devforum.okta.com/uploads/oktadev/original/1X/bf54a16b5fda189e4ad2706fb57cbb7a1e5b8deb.png" href='https://devforum.okta.com/' width="200px"/>
```
Comment (mraible) [inline README.md]: I like the big logo, you don't?

File: README.md [hunk 2]
```diff
@@ -1,26 +1,25 @@
...
-# Okta Developer 'Hub
-
-Okta developer site [developer.okta.com](https://developer.okta.com) is a custom [Jekyll](http://jekyllrb.com/) site deployed on [GitHub pages](https://pages.github.com/).
+The Okta developer site ([developer.okta.com](https://developer.okta.com)) is built using [Jekyll](http://jekyllrb.com/).
 
 All documentation submissions are welcome. To submit a change, please follow the [contribution process](https://github.com/okta/okta.github.io/wiki/Contributing-to-the-Site#you-are-making-a-non-blog-change).
 
+## Getting help
+
+If you have questions or need help with Okta's APIs or SDKs, visit the [Dev Forum](https://devforum.okta.com/) or email developers@okta.com.
```
Comment (mraible) [inline README.md]: I'd mention that emailing developers@okta.com creates a support ticket. Use Stack Overflow or Dev Forum for questions.

File: README.md [hunk 3]
```diff
@@ -1,26 +1,25 @@
...
+
 ## Resources
 - [API Reference](https://developer.okta.com/docs/api/resources/)
+- [SDK Reference](https://developer.okta.com/documentation/)
 - [Developer Blog](https://developer.okta.com/blog/)
-- [Sample Code](https://developer.okta.com/code/)
-- [Contributing to the Site](https://github.com/okta/okta.github.io/wiki/Contributing-to-the-Site)
```
Comment (mraible) [inline README.md]: Why did you remove this link?

File: README.md (hunk 1 above)
Comment (nbarbettini) [reply README.md]: No, I think it looked a little goofy myself. Happy to compromise though 😄

File: README.md (hunk 3 above)
Comment (nbarbettini) [reply README.md]: * Sample Code is a section that we deprecated on the site. (We don't have a single place for code samples now, we have people go to their language of choice first)
* Do you think that the Contributing link is helpful, even though it duplicates the info in the note below it?

Comment (jmelberg-okta) [inline README.md]: Both of these links are important.

1. Sample code if different than SDK reference documentation. We should either group these together under the `/documentation` link, or split them into two different links with a better description.
2. Contributing to the site will get lost - so we need to highlight that here.

**Comment by nbarbettini** [comment]
Updated based on feedback, thanks y'all!

@jmelberg-okta Is there somewhere specific you think we should send folks looking for sample code? The best fit right now IMO is the `/documentation` landing page.

**Comment by jmelberg-okta** [comment]
I think adding a new reference section would make a lot of sense here to link directly to our quickstars. Possibly:
- [Authentication Quick Start](https://developer.okta.com/quickstart/)

**Comment by nbarbettini** [comment]
Thanks for the helpful feedback folks 😄

**Pull request merged**